src/main/java/core/basesyntax/exception/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exception;
+
+public class RegistrationException extends Exception {
```

Comment:
The RegistrationException should extend RuntimeException instead of Exception, since the task requires a custom unchecked exception.

Reply:
Done